pkg/npm/eslint-config/index.js

```diff
@@ -1,204 +1,208 @@
 const env = {
-  "browser": true,
```

Comment:
Imagine it's best to revert the changes to this file before merging this PR, but at least we know it works. 😌